environment/env.go

```diff
@@ -70,75 +70,61 @@ const (
 	PostgresDefaultPort = 5432
 )
 
-// SetupEnv setup the necessary env
-func SetupEnv() {
-	if os.Getenv(LocalhostIP) == "" {
-		err := os.Setenv(LocalhostIP, lookupLocalhostIP("localhost"))
-		if err != nil {
-			panic(fmt.Sprintf("error setting env %v", LocalhostIP))
-		}
-	}
-
-	if os.Getenv(CassandraSeeds) == "" {
-		err := os.Setenv(CassandraSeeds, LocalhostIP)
-		if err != nil {
-			panic(fmt.Sprintf("error setting env %v", CassandraSeeds))
-		}
-	}
-
-	if os.Getenv(CassandraPort) == "" {
-		err := os.Setenv(CassandraPort, strconv.Itoa(CassandraDefaultPort))
-		if err != nil {
-			panic(fmt.Sprintf("error setting env %v", CassandraPort))
-		}
-	}
-
-	if os.Getenv(MySQLSeeds) == "" {
-		err := os.Setenv(MySQLSeeds, LocalhostIP)
-		if err != nil {
-			panic(fmt.Sprintf("error setting env %v", MySQLSeeds))
-		}
-	}
-
-	if os.Getenv(MySQLPort) == "" {
-		err := os.Setenv(MySQLPort, strconv.Itoa(MySQLDefaultPort))
-		if err != nil {
-			panic(fmt.Sprintf("error setting env %v", MySQLPort))
-		}
-	}
-
-	if os.Getenv(PostgresSeeds) == "" {
-		err := os.Setenv(PostgresSeeds, LocalhostIP)
-		if err != nil {
-			panic(fmt.Sprintf("error setting env %v", PostgresSeeds))
-		}
-	}
-
-	if os.Getenv(PostgresPort) == "" {
-		err := os.Setenv(PostgresPort, strconv.Itoa(PostgresDefaultPort))
-		if err != nil {
-			panic(fmt.Sprintf("error setting env %v", PostgresPort))
-		}
-	}
-
-	if os.Getenv(ESSeeds) == "" {
-		err := os.Setenv(ESSeeds, LocalhostIP)
-		if err != nil {
-			panic(fmt.Sprintf("error setting env %v", ESSeeds))
-		}
-	}
+type varSpec struct {
+	Name    string
+	Default func() string
```

Comment:
nit: make these private

Reply:
Done! `Default` is now `getDefault` as `default` is a reserved word